src/components/Header.tsx

```diff
@@ -1,33 +1,65 @@
-import React from "react";
+import React, { useState } from "react";
 import { useCookies } from "react-cookie";
 
-import { Typography, Layout, Button } from "antd";
+import { Typography, Layout, Button, Modal } from "antd";
 
 import logo from "../assets/images/logo.svg";
 import "../styles/Header.css";
 import { withRouter, RouteComponentProps } from "react-router-dom";
 
 const Header: React.FC<RouteComponentProps> = (props) => {
-  const [ cookies, setCookie ] = useCookies(['accessToken']);
+  const [cookies, setCookie] = useCookies(["accessToken"]);
+  const [showExitModal, setShowExitModal] = useState(false);
 
   const onLogoutClick = () => {
-    setCookie('accessToken', '');
+    setCookie("accessToken", "");
     props.history.push("/");
-  }
+  };
+
+  const onLogoClick = () => {
+    console.log("Click");
+    // If the user is NOT logged in, it should redirect them to the homepage
+    if (!cookies["accessToken"]) {
+      props.history.push("/");
+    } else {
+      // If the user is logged in and playing a game, it should show a confirmation dialog
+      console.log("Help");
+      console.log(props.match.path);
+      if (props.match.path == "/DisplayCards") {
+        setShowExitModal(true);
+      } else {
+        props.history.push("/Dashboard");
+      }
+    }
+  };
+
+  const handleOk = () => {
+    setShowExitModal(false);
+    props.history.push("/Dashboard");
+  };
+
+  const handleCancel = () => {
+    setShowExitModal(false);
+  };
 
   return (
     <Layout.Header className="Header">
-      <div className="Logo-Container">
+      <div className="Logo-Container" onClick={onLogoClick}>
```

Comment:
Could you style the Logo-Container so that it shows a pointer cursor on hover. Atm can't tell that it's a button.